Project/Gem/Source/Scripting/LifterController.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root
+ * of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Component/Component.h>
+#include <rclcpp/subscription.hpp>
+#include <std_msgs/msg/bool.hpp>
+#include <std_msgs/msg/detail/bool__struct.hpp>
+
+namespace ROS2::Demo
+{
+    class LifterControllerComponent : public AZ::Component
+    {
+    public:
+        AZ_COMPONENT(LifterControllerComponent, { "80a906af-eacd-4044-94a7-f7f02100fc2a" });
+
+        LifterControllerComponent() = default;
+        ~LifterControllerComponent() = default;
+
+        static void Reflect(AZ::ReflectContext* context);
+
+        static void GetRequiredServices(AZ::ComponentDescriptor::DependencyArrayType& required);
+
+        void Activate() override;
+        void Deactivate() override;
+
+    private:
+        rclcpp::Subscription<std_msgs::msg::Bool>::SharedPtr m_lifterTopicSubscriber;
+
+        AZStd::string m_topic = "example_topic";
```

Comment:
Why the topic is AZStd::string, only ? What about QoS?. We have class for topic  parameters.